md/src/main/java/org/cogcomp/md/MentionAnnotator.java

```diff
@@ -158,6 +161,9 @@ public void addView(TextAnnotation ta) throws AnnotatorException{
         if (!isInitialized()){
             doInitialize();
         }
+        if (!ta.hasView(ViewNames.POS)){
+            ta.addView(posAnnotator);
+        }
```

Comment:
Actually I don't like this. This way we're consuming unnecessary about of memory. 
The right way to do this is like this: 
https://github.com/CogComp/cogcomp-nlp/blob/b8666acb442343a8863f409cd448dceda51d6efe/commasrl/src/main/java/edu/illinois/cs/cogcomp/comma/CommaLabeler.java#L73-L76